app/src/config_constants.js

```diff
@@ -1,8 +1,11 @@
 const LOCAL_TEST_ENVIRONMENT = false;
 
+// set the test server url and port here; either remote hosted staging server or local server
+const testServerUrl = 'http://localhost:3000';
+
 module.exports = Object.freeze({
-    APPLICATION_URL: LOCAL_TEST_ENVIRONMENT ? 'http://localhost:3000' : 'https://www.unitandem.fi',
+    APPLICATION_URL: LOCAL_TEST_ENVIRONMENT ? testServerUrl : 'https://www.unitandem.fi',
     PORT_IN_USE : LOCAL_TEST_ENVIRONMENT ? ':3000' : '',
     IS_LOCAL_TEST_ENV: LOCAL_TEST_ENVIRONMENT,
-    APPLICATION_NAME: 'Unitandem'
+    APPLICATION_NAME: 'Unitandem',
```

Comment:
This is a convention oriented use. Both airbnb and google's style guide for js code supports this. Since, it does not produce any error in the code (but trailing comma on function parameters will do), one good reason to do this is, when you add a new line after some line that has trailing comma, in your pull request, it will only show one line change, basically one line added but if you have to add the comma and an new line then it will be three lines of change the reviewers have to review. So, trailing comma eliminates the extra noise.